src/App.tsx

```diff
@@ -1,61 +1,126 @@
 import './App.scss';
+import { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/TODO';
+
+const idForNewTodo = Math.max(...todosFromServer.map(todo => todo.id)) + 1;
+
+export function getUserById(userId: number) {
+  return usersFromServer.find(user => user.id === userId) || null;
```

Comment:
no needs to return `null` since if you don't find user you needed you'll get falsy value anyway